Mantle/MTLJSONAdapter.m

```diff
@@ -370,13 +370,9 @@ + (NSDictionary *)valueTransformersForModelClass:(Class)modelClass {
 	for (NSString *key in [modelClass propertyKeys]) {
 		SEL selector = MTLSelectorWithKeyPattern(key, "JSONTransformer");
 		if ([modelClass respondsToSelector:selector]) {
-			NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[modelClass methodSignatureForSelector:selector]];
-			invocation.target = modelClass;
-			invocation.selector = selector;
-			[invocation invoke];
-
-			__unsafe_unretained id transformer = nil;
-			[invocation getReturnValue:&transformer];
+			IMP imp = [modelClass methodForSelector:selector];
+			NSValueTransformer*(*function)(id, SEL) = (void *)imp;
```

Comment:
> This (void *) cast is undefined behavior.

Not undefined in this use – a pointer is a pointer. This just silences the compiler... however it _was_ a lazy thing to write and I will fix it!

On the other comment, sure thing!